Client/Frontend/Browser/FindInPageBar.swift

```diff
@@ -56,43 +56,43 @@ class FindInPageBar: UIView {
     override init(frame: CGRect) {
         super.init(frame: frame)
 
-        backgroundColor = UIColor.whiteColor()
+        backgroundColor = UIColor.white()
 
-        searchText.addTarget(self, action: #selector(FindInPageBar.SELdidTextChange(_:)), forControlEvents: UIControlEvents.EditingChanged)
+        searchText.addTarget(self, action: #selector(FindInPageBar.SELdidTextChange(_:)), for: UIControlEvents.editingChanged)
         searchText.textColor = FindInPageUX.SearchTextColor
         searchText.font = FindInPageUX.SearchTextFont
-        searchText.autocapitalizationType = UITextAutocapitalizationType.None
-        searchText.autocorrectionType = UITextAutocorrectionType.No
+        searchText.autocapitalizationType = UITextAutocapitalizationType.none
+        searchText.autocorrectionType = UITextAutocorrectionType.no
         if #available(iOS 9.0, *) {
             searchText.inputAssistantItem.leadingBarButtonGroups = []
             searchText.inputAssistantItem.trailingBarButtonGroups = []
         }
         searchText.enablesReturnKeyAutomatically = true
-        searchText.returnKeyType = .Search
+        searchText.returnKeyType = .search
         addSubview(searchText)
 
         matchCountView.textColor = FindInPageUX.MatchCountColor
         matchCountView.font = FindInPageUX.MatchCountFont
-        matchCountView.hidden = true
+        matchCountView.isHidden = true
         addSubview(matchCountView)
 
-        previousButton.setImage(UIImage(named: "find_previous"), forState: UIControlState.Normal)
-        previousButton.setTitleColor(FindInPageUX.ButtonColor, forState: UIControlState.Normal)
+        previousButton.setImage(UIImage(named: "find_previous"), for: UIControlState())
```

Comment:
This should be `UIControlState.normal` not UIControlState() as should the other below